core/common/src/main/java/alluxio/conf/RemovedKey.java

```diff
@@ -34,63 +34,68 @@
+      put(Name.SWIFT_API_KEY, V2_0_0_REMOVED + " Use " + PropertyKey.Name.SWIFT_PASSWORD_KEY + " "
           + "instead.");
-      put(Name.USER_FAILED_SPACE_REQUEST_LIMITS, V2_REMOVED);
-      put(Name.USER_FILE_CACHE_PARTIALLY_READ_BLOCK, V2_REMOVED);
-      put(Name.USER_FILE_SEEK_BUFFER_SIZE_BYTES, V2_REMOVED);
-      put(Name.USER_HOSTNAME, V2_REMOVED + " Use "
+      put(Name.USER_FAILED_SPACE_REQUEST_LIMITS, V2_0_0_REMOVED);
+      put(Name.USER_FILE_CACHE_PARTIALLY_READ_BLOCK, V2_0_0_REMOVED);
+      put(Name.USER_FILE_SEEK_BUFFER_SIZE_BYTES, V2_0_0_REMOVED);
+      put(Name.USER_HOSTNAME, V2_0_0_REMOVED + " Use "
           + PropertyKey.Template.LOCALITY_TIER.format("node")
           + " instead to set the client hostname.");
-      put(Name.USER_NETWORK_SOCKET_TIMEOUT, V2_REMOVED);
-      put("alluxio.security.authentication.socket.timeout", V2_REMOVED);
-      put("alluxio.security.authentication.socket.timeout.ms", V2_REMOVED);
-      put(Name.USER_RPC_RETRY_MAX_NUM_RETRY, V2_REMOVED);
-      put(Name.MASTER_RETRY, V2_REMOVED);
-      put(Name.USER_UFS_DELEGATION_READ_BUFFER_SIZE_BYTES, V2_REMOVED);
-      put(Name.USER_UFS_DELEGATION_WRITE_BUFFER_SIZE_BYTES, V2_REMOVED);
+      put(Name.USER_NETWORK_SOCKET_TIMEOUT, V2_0_0_REMOVED);
+      put("alluxio.security.authentication.socket.timeout", V2_0_0_REMOVED);
+      put("alluxio.security.authentication.socket.timeout.ms", V2_0_0_REMOVED);
+      put(Name.USER_RPC_RETRY_MAX_NUM_RETRY, V2_0_0_REMOVED);
+      put(Name.MASTER_RETRY, V2_0_0_REMOVED);
+      put(Name.USER_BLOCK_WORKER_CLIENT_THREADS, V2_0_0_REMOVED);
+      put(Name.USER_HEARTBEAT_INTERVAL, V2_0_0_REMOVED);
+      put(Name.USER_UFS_DELEGATION_READ_BUFFER_SIZE_BYTES, V2_0_0_REMOVED);
+      put(Name.USER_UFS_DELEGATION_WRITE_BUFFER_SIZE_BYTES, V2_0_0_REMOVED);
+      put(Name.WORKER_BLOCK_THREADS_MAX, V2_0_0_REMOVED);
+      put(Name.WORKER_BLOCK_THREADS_MIN, V2_0_0_REMOVED);
+      put(Name.WORKER_DATA_BIND_HOST, V2_0_0_REMOVED);
+      put(Name.WORKER_DATA_PORT, V2_0_0_REMOVED);
       put(Name.WORKER_TIERED_STORE_LEVEL0_RESERVED_RATIO, "Use alluxio.worker.tieredstore.levelX"
           + ".watermark.{high/low}.ratio instead");
       put(Name.WORKER_TIERED_STORE_LEVEL1_RESERVED_RATIO, "Use alluxio.worker.tieredstore.levelX"
           + ".watermark.{high/low}.ratio instead");
       put(Name.WORKER_TIERED_STORE_LEVEL2_RESERVED_RATIO, "Use alluxio.worker.tieredstore.levelX"
           + ".watermark.{high/low}.ratio instead");
-      put(Name.WORKER_TIERED_STORE_RETRY, V2_REMOVED);
+      put(Name.WORKER_TIERED_STORE_RETRY, V2_0_0_REMOVED);
 
       put(Name.TEST_REMOVED_KEY, "This key is used only for testing. It is always removed");
     }
   };
 
   static final class Name {
-    public static final String LOGSERVER_HOSTNAME = "alluxio.logserver.hostname";
-    public static final String LOGSERVER_LOGS_DIR = "alluxio.logserver.logs.dir";
-    public static final String MASTER_ACTIVE_UFS_SYNC_BATCH_INTERVAL =
```

Comment:
this and the following one were not in 1.8 but only introduced in 2.0-preview, IMO we don't need to annotate these but only the ones deleted from 1.8